lib/reporters/base.js

```diff
@@ -166,6 +166,25 @@ function stringifyDiffObjs (err) {
   }
 }
 
+/**
+ * Returns a diff between 2 strings with coloured ANSI output.
+ *
+ * The diff will be either inline or unified dependant on the value
+ * of `Base.inlineDiff`.
+ *
+ * @api private
```

Comment:
I think the intent is that this is a public function, so you can remove this, right?